jtune.py

```diff
@@ -1753,6 +1756,41 @@ def _at_exit(raw_gc_log=None, jmap_data=None, jstat_data=None,
                   optimized_for_ygcs_rate)
 
 
+def _generate_rotated_gc_log_name(gc_log_file, log_number):
+    return gc_log_file + "." + str(log_number)
+
+def _get_rotated_log_file(gc_log_file):
+    """Function will scan existing log files to determine latest rotated log, if none found will return
+       non rotated file name.
+    """
+    log_number = 0
+    while os.path.isfile(_generate_rotated_gc_log_name(gc_log_file, log_number)):
+        log_number += 1
+
+    if log_number == 0:
```

Comment:
Better to change this to "if not log number". Zero is false, anything else is true (https://docs.python.org/2/library/stdtypes.html#truth-value-testing)